packages/lib/models/Setting.ts

```diff
@@ -1384,7 +1384,28 @@ class Setting extends BaseModel {
 				label: () => _('Ignore TLS certificate errors'),
 				storage: SettingStorage.File,
 			},
-
+			'sync.proxyUrl': {
```

Comment:
Both these settings should probably be under net.proxyUrl / net.proxyTimeout because it doesn't only applies to sync